src/ui/components/todomvc-app/component.ts

```diff
@@ -1,30 +1,26 @@
 import Component, { tracked } from "@glimmer/component";
 import Navigo from 'navigo';
+import StorageService from '../../../utils/storage';
```

Comment:
fully qualified path like what? 
All I could find based on my research was this reference https://github.com/Microsoft/TypeScript/issues/5039 with baseUrl prop on tsconf.json